announcing-python-sdk-ga.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: 'Announcing General Availability of the Python SDK'
+description: 'We are excited to ship the first generally available version of the Xata Python SDK.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/xata-go-logos.png
+  alt: Xata Python SDK
+author: Philip Krauss
+date: 08-29-2023
+published: true
+slug: announcing-the-python-sdk-ga
+---
+
+At the start, Xata's initial focus was to make things better for developers using [Jamstack](https://chat.openai.com/blog/jamstack-mern-lamp-stack-comparison#jamstack-and-databases). This led to us creating content that predominantly focused on providing a robust [TypeScript offering](https://www.npmjs.com/package/@xata.io/client). 
```

Comment:
This link doesn't seem to work, is it meant to be https://xata.io/blog/jamstack-mern-lamp-stack-comparison#jamstack-and-database?

Also, should it be "using the Jamstack" instead of Jamstack?

Reply:
fixed the link